modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/codec/IntMax.kt

```diff
@@ -10,7 +10,8 @@ enum class IntMax(val max: BigInteger) {
     U128(BigInteger.valueOf(2).pow(128) - BigInteger.ONE),
     U256(BigInteger.valueOf(2).pow(256) - BigInteger.ONE),
     I128(BigInteger.valueOf(2).pow(127) - BigInteger.ONE),
-    I256(BigInteger.valueOf(2).pow(255) - BigInteger.ONE);
+    I256(BigInteger.valueOf(2).pow(255) - BigInteger.ONE),
```

Comment:
omg, even here

Reply:
I know, but I'd rather not fight the linter